lib/base-compiler.js

```diff
@@ -787,7 +787,8 @@ export class BaseCompiler {
     runExecutable(executable, executeParameters) {
         const maxExecOutputSize = this.env.ceProps('max-executable-output-size', 32 * 1024);
         // Hardcoded fix for #2339. Ideally I'd have a config option for this, but for now this is plenty good enough.
-        executeParameters.additionalEnv = {
+        executeParameters.env = {
+            ...executeParameters.env,
```

Comment:
maybe the span should come after? so that executeParameters.env can override these hard-coded values?

Reply:
Works for me, yup